doctr/models/modules/vision_transformer/pytorch.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2022, Mindee.
+
+# This program is licensed under the Apache License 2.0.
+# See LICENSE or go to <https://opensource.org/licenses/Apache-2.0> for full license details.
+
+from typing import Tuple
+
+import torch
+from torch import nn
+
+from ..transformer.pytorch import MultiHeadAttention, PositionwiseFeedForward
+
+__all__ = ["VisionTransformer"]
+
+
+class PatchEmbedding(nn.Module):
+    """Compute 2D patch embedding"""
+
+    # Inpired by: https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/layers/patch_embed.py
```

Comment:
Just FYI: can you confirm that you made a lot of modifications?
inspired by is rather light
borrowed from is more significant